docker/docker-cheribuild.json

```diff
@@ -0,0 +1,7 @@
+# NOTE: Change the paths below to match the locations for your
+# sources, build and output directories.
+{
+  "source-root": "/home/USER/MYDIR/sources",
+  "build-root": "/home/USER/MYDIR/build",
+  "output-root": "/home/USER/MYDIR/output"
+}
```

Comment:
None of this should be needed? The only thing you need is for (docker)-cheribuild outside of Docker to know where source-root is, which is just `{ "source-root": ".." }`, but that file needs to go in the top-level cheribuild directory to be picked up, not here. Everything else should follow automatically if you use a standard cheribuild layout rather than changing things to match your personal preferences.